includes/admin/class-admin.php

```diff
@@ -0,0 +1,218 @@
+<?php
+/**
+ * Admin class for HM TOC plugin.
+ * Adds admin settings for HM TOC plugin to specify default TOC title and header elements list.
+ */
+namespace HM\Content_TOC;
+
+// Abort - if this file is accessed directly and not via WP
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+class Admin {
+
+	// Admin page slug for TOC plugin settings
+	protected $page_slug;
+
+	// Option slug for TOC plugin settings
+	protected  $option_slug;
+
+	// Admin page title
+	protected $page_title;
+
+	/**
+	 * Creates admin object and implements registered actions:
+	 * 1) adds option submenu page to WP Settings page
+	 * 2) sets up plugin settings and displays admin page content
+	 */
+	protected function __construct() {
+
+		// Setup properties used throughout this class
+		$this->page_slug   = 'hm-toc-settings';
+		$this->option_slug = 'hm_content_toc';
+		$this->page_title  = __( 'HM Content TOC Settings', 'hm-content-toc' );
```

Comment:
Erm, haven't read up on translation business yet.. So it means I can't have it as a variable and need to type that multiple times whenever I use that string as a param?